src/actions/app.js

```diff
@@ -1,7 +1,8 @@
 import axios from 'axios';
-
+import { removeDeprecatedFiltersFromState } from '../util/codemirror-utils';
```

Comment:
My biggest issue is with below, but you can definitely come up with a name that is as descriptive and less verbose.